examples/userapi/07_functions_on_subdomains.ipynb

```diff
@@ -0,0 +1,3027 @@
+{
```

Comment:
<p>SAme. All plot should be for the full same [0., 10] x [0, 10] to avoid confusion on where things get computed and defined</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>